src/lib/helpers/generateKeyPair.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { KeyPair } from "@fluencelabs/fluence";
+
+import type { ConfigKeyPair } from "../configs/keyPair";
+
+export const generateKeyPair = async (name: string): Promise<ConfigKeyPair> => {
+  const keyPair = await KeyPair.randomEd25519();
+  return {
+    peerId: keyPair.Libp2pPeerId.toB58String(),
```

Comment:
This functionality should really belong to the `KeyPair` package. I'm not sure how and well we should do it, but we should :)